packages/core/src/oidc/grants/token-exchange/index.ts

```diff
@@ -103,7 +103,9 @@ export const buildHandler: (
     clientId: client.clientId,
     gty: GrantType.TokenExchange,
     client,
-    grantId,
+    // The token exchange grant type does not have a grant ID or grant object,
+    // so we use an empty string here.
+    grantId: '',
```

Comment:
nit: can we omit the grantId field here?

Reply:
No, it is required in `AccessToken` class.